kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -84,9 +84,9 @@ def __init__(self, name, init_config, instances):
         generic_instances = [kubernetes_state_instance]
         super(KubernetesState, self).__init__(name, init_config, instances=generic_instances)
 
-        self.condition_to_status_positive = {'true': self.OK, 'false': self.CRITICAL, 'unknown': self.UNKNOWN}
+        self.condition_to_status_positive = {'true': self.OK, 'false': self.CRITICAL, 'unknown': self.WARNING}
 
-        self.condition_to_status_negative = {'true': self.CRITICAL, 'false': self.OK, 'unknown': self.UNKNOWN}
+        self.condition_to_status_negative = {'true': self.CRITICAL, 'false': self.OK, 'unknown': self.WARNING}
```

Comment:
this map is used in the other checks besides `node.ready`, i think we may want to keep those `unknown` labels mapped to `unknown`? but i don't have the full context of why `out of disk` would be `unknown` cc @clamoriniere